packages/console/src/console-history/ConsoleHistory.tsx

```diff
@@ -15,6 +15,7 @@ interface ConsoleHistoryProps {
   disabled?: boolean;
   supportsType: (type: string) => boolean;
   iconForType: (type: string) => ReactElement;
+  handleCommandSubmit: (command: string) => void;
```

Comment:
The prop event should be named `on*`, whereas the functions that handle the events should be named `handle*`.
See https://react.dev/learn/responding-to-events
> By convention, it is common to name event handlers as handle followed by the event name. You’ll often see onClick={handleClick}, onMouseEnter={handleMouseEnter}, and so on.